Assets/Scripts/Dialogue/InteractiveNPC.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class InteractiveNPC : MonoBehaviour
+{
+    private bool IsConversationActive;
+    private bool IsPlayerInRange;
+    private float DistanceBetweenPlayer;
+    public float MinDistance = 3.5f;
```

Comment:
Forgot this boi; should it be a const as well?

Reply:
No, this one is editable on the editor so the level designer can change the minimum distance according to their needs